src/frontend/planner_test/tests/testdata/output/delete.yaml

```diff
@@ -32,3 +32,14 @@
     create table t (a int, b int);
     delete from t returning sum(a);
   binder_error: 'Bind error: should not have agg/window in the `RETURNING` list'
+- name: distributed delete
+  sql: |
+    set batch_enable_distributed_dml = true;
+    create table t (a int, b int);
+    delete from t;
+  batch_distributed_plan: |-
+    BatchSimpleAgg { aggs: [sum()] }
+    └─BatchExchange { order: [], dist: Single }
+      └─BatchDelete { table: t }
```

Comment:
This reminds me of an issue in ancient times 🤣

https://github.com/risingwavelabs/risingwave/issues/2678

Reply:
🤣